src/github.com/matrix-org/dendrite/clientapi/auth/storage/devices/devices_table.go

```diff
@@ -49,30 +51,32 @@ CREATE UNIQUE INDEX IF NOT EXISTS device_localpart_id_idx ON device_devices(loca
 `
 
 const insertDeviceSQL = "" +
-	"INSERT INTO device_devices(device_id, localpart, access_token, created_ts) VALUES ($1, $2, $3, $4)"
+	"INSERT INTO device_devices(device_id, localpart, access_token, created_ts, display_name) VALUES ($1, $2, $3, $4, $5)"
 
 const selectDeviceByTokenSQL = "" +
-	"SELECT device_id, localpart FROM device_devices WHERE access_token = $1"
+	"SELECT device_id, localpart, display_name FROM device_devices WHERE access_token = $1"
 
 const selectDeviceByIDSQL = "" +
-	"SELECT created_ts FROM device_devices WHERE localpart = $1 and device_id = $2"
+	"SELECT display_name FROM device_devices WHERE localpart = $1 and device_id = $2"
```

Comment:
Where are these changes being used?

Reply:
I wrote the created_ts line originally. I just had to fetch something in order to establish existence. display_name is much  more useful to return when querying for a specific device.